tipg/sql/dbcatalog.sql

```diff
@@ -176,7 +184,7 @@ BEGIN
         jsonb_agg(json_strip_nulls(json_build_object(
             'name', proargname,
             'type', argtype,
-            'default', regexp_replace(def, '''([a-zA-Z0-9_\-\.]+)''::\w+', '\1', 'g')
```

Comment:
This expression was not getting the defaults for functions populated correctly. Replaced this regex with the tipg_fun_defaults function.